### ISSUE TITLE
Require sample to be selected

### ISSUE THREAD

**Issue by naegelyd** [chop-dbhi/varify#64]
There should not be any calls to the `preview` endpoint on the server unless the user has selected a sample. Eventually this might be a config option in Cilantro but until that is fully implemented and thought out, we can just block the calls in Varify if no sample is selected in the user's current filter set.

**Comment by naegelyd** [comment]
This would require an obscene amount of integration of cilantro code into varify itself so I am closing this in favor of making this a config option in Cilantro as requested [here](https://github.com/cbmi/cilantro/issues/313).

**Comment by naegelyd** [comment]
Reopening this as there will still a be a little work to do(in the config file) to mark the sample concept as required. We will just be leveraging the new config options in Cilantro once they are available. This issue is blocked by https://github.com/cbmi/cilantro/issues/313 and its related issues.
